### PR TITLE
Showing the drawing and sketch functionality in QField

### PR DESCRIPTION
**Pull request by SeqLaz**
Added documentation showing the drawing and sketch functionality in QField

https://github.com/opengisch/QField/pull/5149

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-4039 #5149 Implementation of basic image attachment sketching capability](https://app.clickup.com/t/2192114/QF-4039)

File: documentation/how-to/pictures.en.md [hunk 1]
```diff
@@ -66,6 +66,12 @@ Set the widget type of the field path to *Attachment* and add it to the form
 
 !![widgets](../assets/images/add-1-n-pictures-widgets_picture.png)
 
+## Drawing and Sketching Functionality
+
+QField now has an in-app drawing and sketching functionality, this feature enables to sketch directly over images taken within the application, enhancing data annotation capabilities. The functionality is seamlessly integrated into the attachment widget, allowing users to quickly annotate images with sketches and providing an option to undo sketches if needed for corrections 😉.
```
Comment (nirvn) [inline documentation/how-to/pictures.en.md:71]: I don't think we need to mention the undo functionality; it's self-evident in the screencast, and I think it's something people would expect in any case :)

```suggestion
QField now has an in-app drawing and sketching functionality enabling you to directly sketch over and annotate images captured while in the field as well as drawing on top of a blank canvas or over a template. The functionality is seamlessly integrated into the feature form's attachment widget.
```

Comment (m-kuhn) [reply documentation/how-to/pictures.en.md:71]: Should we drop the "now"? This functionality will soon not be so new anymore

Comment (nirvn) [reply documentation/how-to/pictures.en.md:71]: @m-kuhn , yep.

File: documentation/how-to/pictures.en.md [hunk 2]
```diff
@@ -66,6 +66,12 @@ Set the widget type of the field path to *Attachment* and add it to the form
 
 !![widgets](../assets/images/add-1-n-pictures-widgets_picture.png)
 
+## Drawing and sketching
+
+QField have an in-app drawing and sketching functionality enabling you to directly sketch over and annotate images captured while in the field as well as drawing on top of a blank canvas or over a template. The functionality is seamlessly integrated into the feature form's attachment widget.
```
Comment (suricactus) [inline documentation/how-to/pictures.en.md:71]:
```suggestion
QField have an in-app drawing and sketching functionality enabling you to directly sketch over and annotate images captured while in the field as well as drawing on top of a blank canvas or over a template.
```
No strong opinion against it, but I feel like this sentence is more marketing than documentation. IMO it does not bring documentation value.

**Review by suricactus — APPROVED** [review]

File: documentation/how-to/pictures.en.md (hunk 2 above)
Comment (SeqLaz) [reply documentation/how-to/pictures.en.md]: Reading well, you are right!

**Comment by suricactus** [comment]
Please squash and merge or force push with the squashed version before merging, otherwise we will have 40MB GIF in the git history.

**Comment by SeqLaz** [comment]
Hey @suricactus and @nirvn I just finished the squash and wanted to confirm if I should proceed in the same way with the other open PR that has similar files?

**Comment by nirvn** [comment]
Soon to be out there as part of QField 3.3 :heart:

**Pull request merged**